src/courtroom.cpp

```diff
@@ -2393,108 +2408,97 @@ void Courtroom::display_pair_character(QString other_charid, QString other_offse
           }
         }
       }
-    }
-    else if (!ui_vp_chatbox->set_chatbox(chatbox_path)) {
-      chatbox_path = ao_app->get_theme_path("chat");
-      if (!ui_vp_chatbox->set_chatbox(chatbox_path)) {
-        chatbox_path = ao_app->get_theme_path("chatbox");
-        if (!ui_vp_chatbox->set_chatbox(chatbox_path)) {
-          chatbox_path = ao_app->get_default_theme_path("chat");
-          if (!ui_vp_chatbox->set_chatbox(chatbox_path)) {
-            chatbox_path = ao_app->get_default_theme_path("chatbox");
-            ui_vp_chatbox->set_chatbox(chatbox_path);
-          }
-        }
-      }
-    }
-  }
-  else // Aw yeah dude do some showname magic
-  {
-    if (!ui_vp_chatbox->set_image("chat"))
-      ui_vp_chatbox->set_image("chatbox");
-    set_font(ui_vp_showname, "", "showname", customchar);
-    QFontMetrics fm(ui_vp_showname->font());
+      else // Aw yeah dude do some showname magic
+      {
+        if (!ui_vp_chatbox->set_image("chat"))
+          ui_vp_chatbox->set_image("chatbox");
+        set_font(ui_vp_showname, "", "showname", customchar);
+        QFontMetrics fm(ui_vp_showname->font());
 // Gotta support the slow paced ubuntu 18 STUCK IN 5.9.5!!
 #if QT_VERSION > QT_VERSION_CHECK(5, 11, 0)
-    int fm_width = fm.horizontalAdvance(ui_vp_showname->text());
+        int fm_width = fm.horizontalAdvance(ui_vp_showname->text());
 #else
-    int fm_width = fm.boundingRect((ui_vp_showname->text())).width();
+        int fm_width = fm.boundingRect((ui_vp_showname->text())).width();
 #endif
-    QString chatbox_path = ao_app->get_theme_path("chat");
-    QString chatbox = ao_app->get_chat(customchar);
+        QString chatbox_path = ao_app->get_theme_path("chat");
+        QString chatbox = ao_app->get_chat(customchar);
 
-    if (chatbox != "" && ao_app->is_customchat_enabled()) {
-      chatbox_path = ao_app->get_theme_path("misc/" + chatbox + "/chat");
-      if (!ui_vp_chatbox->set_chatbox(chatbox_path)) {
-        chatbox_path = ao_app->get_base_path() + "misc/" + chatbox + "/chat";
-        if (!ui_vp_chatbox->set_chatbox(chatbox_path))
-          ui_vp_chatbox->set_chatbox(chatbox_path + "box");
-      }
-    }
+        if (chatbox != "" && ao_app->is_customchat_enabled()) {
+          chatbox_path = ao_app->get_theme_path("misc/" + chatbox + "/chat");
```

Comment:
I'm... not sure this has to be indented. I suspect it's on the wrong part of the control flow now.

Reply:
For this part it was after adding in missing brackets and trusting my editor to indent. I don't know how the brackets went missing after the merge attempt, but the branch builds now.

Reply:
The client seems to work properly, although Ema's gets to eat 2.9 snackoos instead of 1 before talking. Feature or bug? The chat arrow also moved up too.

Reply:
i can only assume Ema's snackoo eating was truncated by the [Time] part of the char.ini, which after my rewrite no longer does anything

Reply:
the chat_arrow moving up is due to no longer using QMovie to render it, which moves it because Qt is just Like That